### DISCUSSION TITLE
【FTX破產延燒2】比特幣從高點暴跌7成　這位大咖趁勢抄底

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#862] [General]
我覺得我可能這輩子都不會想嘗試進入加密貨幣的市場，雖然有人覺得這類去中心化市場很方便，但我覺得市場本身的資金流動起伏太大，我可能沒這個心臟去接受它今天還正常，明天一起床公司直接破產這種風險!😱

https://www.mirrormedia.mg/story/20221124money002/

**Comment by wangtyphoon** [comment]
目前的數位貨幣市場無疑是場鬱金香狂熱，同樣一筆錢流來流去相互質押，創造虛假的市值

**Comment by williamyen0623** [comment]
如果害怕的話，分個資產的1%~5%去玩玩就好，說不定會有意想不到的回饋，我自己是蠻看好區塊鏈和加密貨幣產業的，多學不虧

**Comment by StevenWanglolz** [comment]
我還聽到一些陰謀論說這是他們一手策畫的，歡迎加入區塊練研究社多多了解哈哈哈

**Comment by CPCpyt** [comment]
我覺得這個事件是非常好的例子，相信可以帶給加密貨幣市場以及整個投資市場相當多的啟示。
也許加密貨幣也會需要一個統一的監督系統保障其市場的穩定以及成長。

**Comment by CPCpyt** [comment]
另外，對於加密貨幣，我現在還無法確切感受到它的價值。
若真要說我認為其價值就是「獲利」本身，它的波動性的確可以運用槓桿帶來豐厚的獲利機會。
我很欣賞他的概念，但我想全球社會還需要很多時間來適應並運用。
